lib/vdsm/network/netinfo/cache.py

```diff
@@ -26,10 +26,10 @@
 
 import six
 
+import vdsm.network.link.iface
 from vdsm.network import nmstate
 from vdsm.network.ip.address import ipv6_supported
-from vdsm.network.ipwrapper import getLinks
-from vdsm.network.link import iface as link_iface
+from vdsm.network import ipwrapper
```

Comment:
`vdsm.network` should be alphabetically before `vdsm.network.ip.addres`.

Reply:
done